app/services/supported_remediations.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+# Service for Supported Remediations
+class SupportedRemediations
+  class << self
+    def revision
+      SsgConfigDownloader.update_ssg_ansible_tasks
```

Comment:
@skateman I realized that this line was needed because without it, this method was returning the revision date in the fallback ansible tasks file `config/ssg-ansible-tasks.default.yaml`. I deployed to ephermal manually and made sure that the remediation revision date was syncing up and everything looked good